documentation/USAGE.rst

```diff
@@ -535,68 +535,223 @@ Further demonstrations of spatially varying boundary condition can be found
 in :mod:`examples.diffusion.mesh20x20`
 and :mod:`examples.diffusion.circle`
 
-Applying internal boundary conditions
-=====================================
+Applying Robin boundary conditions
+==================================
+
+The Robin condition
+
+.. math::
+
+   a\phi + b\hat{n}\cdot\nabla\phi = g\qquad\text{on $f=f_0$}
```

Comment:
Probably use \vec{a} there.

Reply:
Per [wikipedia](https://en.wikipedia.org/wiki/Robin_boundary_condition), it's a scalar equation. For completeness, it should probably be $\hat{n}\cdot\vec{a}$

Reply:
Converted $a$ to $\hat{n}\cdot\vec{a}$ throughout